microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,627 @@
+
+1. Run the downloaded Mixed Reality Feature Tool executable, and follow the prompts to install the Mixed Reality Toolkit (MRTK) and any additional required packages for HoloLens 2 development.
+
+   The Mixed Reality Feature Tool allows you to select and install the appropriate versions of the Mixed Reality Toolkit (MRTK), along with other Mixed Reality features, directly into your Unity project.
+
+After you've installed the Mixed Reality Feature Tool and the Mixed Reality Toolkit (MRTK), the MRTK assets and prefabs appear in your Unity project, ready for use.
+
+Continue to the next step.
+
+
+<!-- ====================================================================== -->
+## Step 4 - Install the WebView plugin for Unity by using the Mixed Reality Feature Tool
+
+In this step, you'll use the Mixed Reality Feature Tool to install the Microsoft Mixed Reality WebView plugin for Unity.  This will install the specific version of the WebView2 NuGet package that's compatible with HoloLens 2.
+
+1. Launch the Mixed Reality Feature Tool (which you installed above). The **Start** page of the Mixed Reality Feature Tool for Unity opens, containing options to configure settings and to view documentation:
+
+   ![The Configure Settings button in the Mixed Reality Feature Tool](./hololens2-images/mrtk_feature_tool_start_page.png)
+
+1. Click the gear button.  The **Settings** pane opens.
+
+1. Click the **Feature** tab, and then make sure the **Show preview releases** checkbox is selected:
+
+   ![the Feature tab in Settings for the Mixed Reality Feature Tool](./hololens2-images/feature_enable_preview_releases.png)
+
+1. Click the **Ok** button to return to the starting page.
+
+1. Click the **Start** button to begin discovering feature packages.
+
+1. Point the Mixed Reality Feature Tool to your Unity project.  To do this, click the **Browse for the project folder** (**...**) button to the right of the **Project Path** field:
+
+   ![Select the Unity project in the Mixed Reality Feature Tool](./hololens2-images/feature_tool_select_unity_project.png)
+
+1. In the file selection dialog, select your Unity project folder, and then click the **Open** button.
+
+1. In the **Select project** dialog, click the **Discover Features** button.
+
+1. In the **Discover Features** page, expand the **Other features** section, and then select the **Microsoft Mixed Reality WebView** checkbox:
+
+   ![The Discover Features pane in the Mixed Reality Feature Tool](./hololens2-images/mrtk_webview2_selected.png)
+
+   This marks the WebView plugin for Unity for installation. In the version dropdown, by default, the latest version of the plugin is selected.
```

Comment:
```suggestion
   This marks the WebView plugin for Unity for installation. In the **Version** dropdown, by default, the latest version of the plugin is selected.
```

Reply:
fixed locally better; added "list"

Reply:
latest Files Changed has bold init cap + "list"
https://github.com/MicrosoftDocs/edge-developer/pull/2567/files

can close this thread